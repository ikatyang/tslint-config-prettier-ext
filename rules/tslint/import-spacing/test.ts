import*as ts from "typescript";
