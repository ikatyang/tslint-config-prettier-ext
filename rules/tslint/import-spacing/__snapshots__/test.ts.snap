// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
import*as ts from \\"typescript\\";

======
import * as ts from \\"typescript\\";

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (import-spacing) /rules/tslint/import-spacing/test.ts[1, 1]: Add space after 'import'
ERROR: (import-spacing) /rules/tslint/import-spacing/test.ts[1, 7]: Add space after '*'
"
`;
